standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/conf/MetastoreConf.java

```diff
@@ -67,6 +67,9 @@ public class MetastoreConf {
   static final String DEFAULT_STORAGE_SCHEMA_READER_CLASS =
       "org.apache.hadoop.hive.metastore.DefaultStorageSchemaReader";
   @VisibleForTesting
+  static final String SERDE_STORAGE_SCHEMA_READER_CLASS =
+      "org.apache.hadoop.hive.metastore.SerDeStorageSchemaReader";
```

Comment:
Since you are introducing this, could you please add assertion case here - Check the below -
https://github.com/apache/hive/blob/c92a478e514a28a53009fe5fbf08ce6fa35b58b9/standalone-metastore/metastore-server/src/test/java/org/apache/hadoop/hive/metastore/conf/TestMetastoreConf.java#L482

Reply:
Added assertion about new class in the test case